tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2020",
```

Comment:
Why is it needed?

Reply:
It was because I was drafting with divisor [BigInt native](https://github.com/gnosis/ido-ux/compare/develop...henrypalacios:issue-469?diff=split#diff-5f88dd7ddd21194834d24abc9a7dcb126132d27605fec3cf6695c8228f9a7b07R235). But I have already replaced it